test/test_write_concern.py

```diff
@@ -0,0 +1,50 @@
+import collections
+import unittest
+
+from pymongo.write_concern import WriteConcern
+
+
+class TestWriteConcern(unittest.TestCase):
+
+    def test_equality(self):
+        concern = WriteConcern(j=True, wtimeout=3000)
+        self.assertEqual(concern, WriteConcern(j=True, wtimeout=3000))
+        self.assertNotEqual(concern, WriteConcern())
+
+    def test_equality_to_none(self):
+        concern = WriteConcern()
+        self.assertTrue(concern != None)  # noqa
```

Comment:
Is there a reason we are not using `self.assertEqual` and `self.assertNotEqual` here? It's not a big deal but personally, I am not a fan of using `self.assertTrue` because it doesn't make the test-writers intent obvious (one must inspect the expression to infer what is being tested).

Reply:
I wanted to explicitly call `!=`, but I've switched to assertNotEqual for the other one (which uses == internally). I've added a comment to make it more obvious.